### ISSUE TITLE
Margin affect the col-widt/row-height

### ISSUE THREAD

**Issue by dronShepherd** [BTMorton/angular2-grid#152]
Hi,

either im configuring wrong or understanding something wrong.
my configuration: 
![image](https://cloud.githubusercontent.com/assets/7661414/19271001/18b4516a-8fc3-11e6-9fe9-8eb36edcff81.png)
![image](https://cloud.githubusercontent.com/assets/7661414/19271033/2f3a49f8-8fc3-11e6-8e33-93ea5b142bd1.png)
this is ok but by increasing the margin to one the with increases by 3:
![image](https://cloud.githubusercontent.com/assets/7661414/19271080/510bea82-8fc3-11e6-80d8-ac835e8b05f1.png)

some ideas?

**Comment by BTMorton** [comment]
Yeah, that's correct. For every width 1 row that you have, adding a margin of 1 makes the total row width 1. That's because the smallest row would be 1px wide with a 1px margin either side. So when the item width gets calculated it's `(width * rowCount) + ((marginLeft + marginRight) * (rowCount - 1))`. Does that make sense?
